dbt_platform_helper/templates/environments/main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  environments = yamldecode(file("../../../platform-config.yml"))["environments"]
```

Comment:
Should this sit under `args` like `services`?

Reply:
Good point. Actually it probaby makes sense to pull the yamldecode out into the locals, and reference "environments" and "extensions" parts in the args. I'll refactor